### PR TITLE
Feature/1 move exasol dialect

### LINKED ISSUE #1: Move Exasol dialect

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**
Closes #1

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,11 +1,19 @@
-target/
-pom.xml.tag
-pom.xml.releaseBackup
-pom.xml.versionsBackup
-pom.xml.next
-release.properties
+/target/
+
+# .settings : we need Eclipse settings for code formatter and clean-up rules
+target
```
Comment (redcatbear) [inline .gitignore]: duplicate target entry.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,65 @@
...
+
+## Features
+
+* Access Exasol database using Virtual Schema.
+
+## Table of Contents
+
+### Information for Users
+
+* [Exasol dialect](doc/dialects/exasol.md)
```
Comment (redcatbear) [inline README.md]: Add links to the central parts of the documentation.

Repeat for developer documentation.

File: doc/dialects/exasol.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
+# Exasol SQL Dialect
+
+Connecting to an Exasol database is the simplest way to get started with Virtual Schemas.
+You don't have to install any JDBC driver, because it is already installed in the Exasol database and also included in the JAR of the JDBC adapter.
```
Comment (redcatbear) [inline doc/dialects/exasol.md]: This reminds me that we can now probably remove the Exasol JDBC driver from the other dialects.

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,325 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.exasol</groupId>
+    <artifactId>exasol-virtual-schema</artifactId>
+    <version>${product.version}</version>
+    <name>Exasol Virtual Schema</name>
+    <description>This projects contains an Exasol dialect of Virtual Schema</description>
```
Comment (redcatbear) [inline pom.xml]:
```suggestion
    <description>This projects contains the Exasol dialect for Exasol's Virtual Schema</description>
```

File: .gitignore (hunk 1 above)
Comment (AnastasiiaSergienko) [reply .gitignore]: removed

File: README.md (hunk 2 above)
Comment (AnastasiiaSergienko) [reply README.md]: done

**Pull request merged**